src/React/Resources/assets/src/render_controller.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import React, { ReactElement } from 'react';
+import { createRoot } from 'react-dom/client';
```

Comment:
Hmm, actually this is proving problematic. I merged, thinking the `yarn build` failure was something unrelated, but it's not. `yarn build` fails here with:

> Error: 'createRoot' is not exported by node_modules/react-dom/client.js, imported by src/React/Resources/assets/src/render_controller.ts

That is not correct... as `createRoot` IS an export of that file. I'm not sure if we may need `commonjs` for rollup (https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module... also ref similar issue: https://github.com/JedWatson/react-select/issues/3956#issuecomment-612104302

Reply:
Looks like I fixed it over here: https://github.com/symfony/ux/pull/330